### ISSUE TITLE
[iOS] Memory leak when file chunks are read in loops

### ISSUE THREAD

**Issue by birdofpreyru** [birdofpreyru/react-native-fs#12]
Hello, @birdofpreyru  it also doesn't close on ios as well. When I read chunks in a loop, the memory consumption just keeps on getting higher and higher. Sadly, I couldn't figure out how to close the file on my own.

_Originally posted by @ISnowFoxI in https://github.com/birdofpreyru/react-native-fs/issues/10#issuecomment-1757589607_

**Comment by sagark1510** [comment]
@birdofpreyru any update on this? I'm not good at swift/obj-c, else I would've raised PR already

**Comment by birdofpreyru** [comment]
No. The issue is marked as _On Hold_, meaning I don't actively (plan to) look into it, at the moment. It does not cause much troubles in my projects, and investing my time into resolving it does not look as the best investment of my time, as of now.

**Comment by sagark1510** [comment]
@birdofpreyru I understand. Thank you.
